src/main/scala/chisel3/aop/Select.scala

```diff
@@ -262,12 +262,15 @@ object Select {
     }
   }
 
-  /** Selects all ios on a given module
+  /** Selects all Data ios on a given module
+    *
+    * Note that Property ios are not returned.
+    *
     * @param module
     */
   def ios(module: BaseModule): Seq[Data] = {
     check(module)
-    module._component.get.asInstanceOf[DefModule].ports.map(_.id)
+    module._component.get.asInstanceOf[DefModule].ports.map(_.id).collect { case (d: Data) => d }
```

Comment:
I chose to filter down to Data here, because this gets used in the `def ios`  below, where it uses `parent._lookup`, and I don't have a typeclass instance for `Lookupable[IsBindable]` yet.

Reply:
i think this is fine for now